internal/service/model_inbound.go

```diff
@@ -176,3 +181,20 @@ type ODFIStorage struct {
 	// RemoveZeroByteFiles determines if we should delete files that are zero bytes
 	RemoveZeroByteFiles bool
 }
+
+type SQSConfig struct {
+	Session  aws.Config
```

Comment:
Is this supposed to be definable in JSON/YAML?

Reply:
I haven't personally tried this yet myself for SQS, but [one neat capability](https://gocloud.dev/concepts/urls/) of gocloud is that it provides a slightly vendor neutral way of doing this too.  The Blob storage [has a good example](https://pkg.go.dev/gocloud.dev/blob/s3blob?utm_source=godoc#example-package-OpenBucketFromURL) of this.

Would the `pubsub` abstraction from gocloud be enough? For the blob storage as an example, you can customize how you want the underlying blob storage to be access by hacking the URL a bit.  The docs [for pubsub](https://pkg.go.dev/gocloud.dev@v0.27.0/pubsub#OpenSubscription) seem to indicate a similar capability.